helm/micro-integrator/README.md

```diff
@@ -0,0 +1,194 @@
+# Helm Chart for deployment of WSO2 Micro Integrator
+
+## Contents
+
+* [Prerequisites](#prerequisites)
+* [Quick Start Guide](#quick-start-guide)
+
+## Prerequisites
+
+* In order to use WSO2 Helm resources, you need an active WSO2 subscription. If you do not possess an active WSO2
+  subscription already, you can sign up for a WSO2 Free Trial Subscription from [here](https://wso2.com/free-trial-subscription)
+  . Otherwise you can proceed with docker images which are created using GA releases.<br><br>
+
+* Install [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), [Helm](https://github.com/kubernetes/helm/blob/master/docs/install.md)
+(and Tiller) and [Kubernetes client](https://kubernetes.io/docs/tasks/tools/install-kubectl/) in order to run the 
+steps provided in the following quick start guide.<br><br>
+
+* An already setup [Kubernetes cluster](https://kubernetes.io/docs/setup/pick-right-solution/).<br><br>
+
+* Install [NGINX Ingress Controller](https://kubernetes.io/docs/setup/). Please note that Helm resources for WSO2 product
```

Comment:
Ingress Controller link is incorrect.
**https://kubernetes.github.io/ingress-nginx/deploy/**